Dockerfile

```diff
@@ -1,22 +1,43 @@
-FROM kbase/sdkbase2 as build
+FROM eclipse-temurin:11-jdk as build
+
+WORKDIR /tmp
+RUN apt-get update -y && \
+        apt-get install -y ant git ca-certificates tzdata python3-sphinx && \
```

Comment:
why tzdata btw?

Reply:
It's gone now, but it was there to prevent issues when installing python - python will prompt you for your timezone, which halts automated builds (e.g. on GHA); installing tzdata first stops that prompting.

Reply:
:+1: